datasets/modeled/dgen/dataset.json

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
It seems that you renamed this file to use a `.json` extension. However, the contents are JSON5 and so dsgrid will break. You can use regular JSON if you want, but you would have to convert the format.

Reply:
Ok was not sure what to use as  the "empty" dataset files Elanie created were json. Does dsgrid accept both?